dashboard/dashboard-front/src/components/XTooltip.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+    <q-tooltip class="bg-primary text-body2">
+        <div style="max-width: 100ch;">
+            <slot />
+        </div>
+    </q-tooltip>
+</template>
```

Comment:
No newline at end of file (I think? Github has this odd symbol here, but inspect element calls it the no newline at end of file symbol.)